code/modules/organs/limbs.dm

```diff
@@ -55,6 +56,9 @@
 	var/vital //Lose a vital limb, die immediately.
 	var/germ_level = 0		// INTERNAL germs inside the organ, this is BAD if it's greater than INFECTION_LEVEL_ONE
 
+	///What % of the body does this limb cover. Make sure that the sum is always 100.
+	var/cover_index = 0
```

Comment:
Just to be clear if you didn't sum up to 100, it would just meant that the armor is less effective or more effective than intended?

For example if chest had 100 cover_index it would be 76% more effective, since its only meant to have a 27 coverage?

Reply:
If the total value is less than 100 then the sum of all armors, in the mob, will be lower than the median among them.

If the chest had a higher cover index it would contribute more towards the total armor in the mob.

Not sure if this answers your question?